Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -86,8 +86,16 @@ private static InputActionAsset CreateNewActionAsset()
                 }
             }
 
-            // Create sub-asset for each action. This is so that users can select individual input actions from the asset when they're
-            // trying to assign to a field that accepts only one action.
+            CreateInputActionReferences(asset);
+
+            AssetDatabase.SaveAssets();
+
+            return asset;
+        }
+
+        private static void CreateInputActionReferences(InputActionAsset asset)
+        {
+            var maps = asset.actionMaps;
```

Comment:
Do we need a check here to make sure the asset doesn't already have the InputActionReferences before they are created?

var actionReference = ScriptableObject.CreateInstance<InputActionReference>();